spec/cell_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'cell'
+
+describe Cell do
+
+  let(:cell){ Cell.new }
+
+  describe '#new' do
```

Comment:
I'm stealing this. Very semantic idea instead of describe initialize. Thanks! ;)